.github/workflows/ci.yml

```diff
@@ -24,12 +24,58 @@ jobs:
         with:
           python-version: ${{ matrix.python_version }}
 
-      # Use our tasks in the playbook which is about as fast.
-      # - name: Set up KinD cluster
-      #   uses: engineerd/setup-kind@v0.3.0
-
       - name: Install ansible and other dependencies.
         run: pip install ansible ansible-lint yamllint docker molecule openshift
 
       - name: Run molecule default test scenario
         run: molecule test
+
+  ansible-acd:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python_version: ['3.7']
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v1
+
+      - name: Set up Python ${{ matrix.python_version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python_version }}
+
+      - name: Install Ansible ACD distribution.
+        run: >-
+          pip install ansible
```

Comment:
Alternatively, use this other deps style, to be more precise: https://github.com/ansible/ansible-lint/blob/master/tox.ini#L18-22.
Otherwise, it's not guaranteed where pip will discover the dist from.

Or at least add `>=2.10` there

Reply:
Thanks, added the version constraint for now.